lib/api/model/model.dart

```diff
@@ -664,6 +665,37 @@ enum MessageFlag {
   String toJson() => _$MessageFlagEnumMap[this]!;
 }
 
+@JsonSerializable(fieldRename: FieldRename.snake)
+class MessageEditHistory {
+
+  final String? prevContent;
+  final String? prevRenderedContent;
+  final int? prevRenderedContentVersion;
+  final int? prevStream;
+  final String? prevTopic;
```

Comment:
The API docs say this field was renamed in Zulip Server 5.0. This codebase currently still supports back to Zulip Server 4.0 — see the "Editing API types" section in the README. So this should handle the old name too.

To avoid the rest of the code needing to look for both names, we should have the class use only the new name, but have the deserialization code look for the old name if it doesn't find the new name. See `User.isSystemBot` for an example of how to do that.

(Then that will also be a case that should definitely get a test.)